### ISSUE TITLE
Tools->Convert Fractions->Unicode Fractions inconsistently misses some

### ISSUE THREAD

**Issue by okrick** [DistributedProofreaders/guiguts-py#384]
The 'Tools -> Convert Fractions -> Unicode Fractions Only' and 'Tools -> Convert Fractions -> Unicode Fractions or Super/Subscript' options inconsistently fail to convert fractions on the first line of a block markup. For example, the fraction '1/4' might remain unchanged despite the conversion settings. This issue does not appear to occur with the 'Tools -> Convert Fractions -> All to Super/Subscript' option."  For example:

```
/*
1/4 POUND CHINESE CURED PORK (Cut into fine shreds.) (See page 4 for recipe for curing pork.)
1/2 CUP BAMBOO SHOOT (Cut into fine, long shreds.)
4 CHINESE WATER CHESNUTS[**P1: CHESTNUTS?] (Pared and cut into fine shreds.)
1/2 CUP CELERY (Cut into fine, long shreds.)
1 STEM OF GREEN ONION TOP (Cut small.)
6 EGGS.
1 TEASPOONFUL CHINESE SALTY SAUCE.
*/
```
sometimes results in 
```
/*
1/4 POUND CHINESE CURED PORK (Cut into fine shreds.) (See page 4 for recipe for curing pork.)
½ CUP BAMBOO SHOOT (Cut into fine, long shreds.)
4 CHINESE WATER CHESNUTS (Pared and cut into fine shreds.)
½ CUP CELERY (Cut into fine, long shreds.)
1 STEM OF GREEN ONION TOP (Cut small.)
6 EGGS.
1 TEASPOONFUL CHINESE SALTY SAUCE.
 */
```

The only thing I found consistent is that the missed fraction is always the list's first item.  But I can't replicate it every time despite repeated tries.  It seems random.

[projectID6579ecdbc1e27.zip](https://github.com/user-attachments/files/16605759/projectID6579ecdbc1e27.zip)

**Comment by windymilla** [comment]
@okrick, I'm sorry - despite repeated attempts, I can't replicate this.

**Comment by srjfoo** [comment]
@okrick, neither can I. I'm not doubting that you've seen an issue with it, but the only way I could get the 1/4 on the first line to not convert was to fail to select the `1` in the fraction. I tried multiple different selections, and multiple times (though not more than 3 to 5 times each):
* Select all
* Selecting the block including the no-wrap markup
* Selecting the block without the markup
* Selecting the line
* Selecting just the character.

**Comment by okrick** [comment]
I appreciate both of you trying to troubleshoot the problem. Unfortunately, the issue was intermittent, and I couldn't identify any specific patterns either between successful and unsuccessful attempts.

**Comment by windymilla** [comment]
Having had another unsuccessful attempt to reproduce, I'm going to close this for now. We can re-open if it strikes again.

**Comment by windymilla** [comment]
Hopefully now fixed by #568
